js/display.js

```diff
@@ -111,7 +111,7 @@ function displayBuildStats(parent_id,build,command_group,stats){
                     style === "positive" ? style = "negative" : style = "positive"; 
                 }
                 if (id === "poison" && id_val > 0) {
-                    id_val = Math.ceil(id_val*stats.get("poisonPct")/100);
+                    id_val = Math.ceil(id_val*(1+stats.get("poisonPct")/100));
```

Comment:
actually I think this is fully incorrect -- displayed value should not depend on curse/courage.. oops